client/elements/addons/sc-top-sheet-views.js

```diff
@@ -332,6 +340,27 @@ export class SCTopSheetViews extends LitLocalized(LitElement) {
     this._fetchReferenceDisplayType();
   }
 
+  // This componenet is _never_ disconnected the way the code is written today (2024-08-04)
+  // Still, adding this in case some day this component is properly unloaded.
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    document.removeEventListener('keydown', this._handleKeydown);
+  }
+
+  _handleKeydown(event) {
+    switch (event.key) {
+      case 'v':
+      case 'V':
+        let idx = this.textViewArray.findIndex(item => item.textView === this.selectedTextView);
+        if (idx >= 0) {
+          idx = (idx + 1) % this.textViewArray.length;
+          this.#setTextBilaraPageIsTextOptionsMismatchSavedSettingsState();
```

Comment:
@ihongda - this is the magic function call I needed to make it work.

Reply:
👍🏼